shared/checks/oval/installed_os_is_ol.xml

```diff
@@ -0,0 +1,24 @@
+<def-group>
+
+  <definition class="inventory" id="installed_os_is_ol" version="1">
+    {{{ oval_metadata("Installed OS is ol", affected_platforms=["multi_platform_all"]) }}}
```

Comment:
Please use capital letters for `ol` - `Installed OS is OL`.
And is there a reason to have filename only lowercase? I see `installed_os_is_rhel.xml` and now `installed_os_is_ol.xml`, but all previous checks have it with capital `OS`, `installed_OS_is_*`.

Reply:
Hello @mildas  I am sorry for this, as I use screenreader I overlooked the capitalization. Capital letters can be detected with screenreader but it is not that obvious as if you see it visually. I will pay attention to that in the future.

Reply:
@vojtapolasek No problem. Another tiny problem now is that `OL` in filename is capital. According to other filenames in `checks/oval`, only `OS` is capital and the rest is lower case.
String in OVAL check is correct.

Reply:
Ach... what to say. Thanks @mildas , fixed.